File: 백준/Bronze/27110. 특식 배부/특식 배부.c
#include <stdio.h>

int main(void) { 
   int a;
   scanf("%d", &a);
   
   int x, y, z;
   scanf("%d %d %d", &x, &y, &z);
   int sum = 0;

   if(x <= a) sum += x;
   else sum += a;
   if(y <= a) sum += y;
   else sum += a;
   if(z <= a) sum += z;
   else sum += a;

   printf("%d", sum);

   return 0;  
}

File: 백준/Bronze/27110. 특식 배부/특식 배부.py
a = int(input())
x, y, z = map(int, input().split())
sum = 0

if x <= a:
  sum += x
else:
  sum += a

if y <= a:
  sum += y
else:
  sum += a

if z <= a:
  sum += z
else:
  sum += a

print(sum)